### PR TITLE
Add Docusaurus message

### PR DESCRIPTION
**Pull request by mikena-truss**
Add a disclaimer/alert for Docusaurus changes to main README

### CODE REVIEW

**Review by rpdelaney — APPROVED** [review]
One non-blocking comment. Thank you for taking the initiative on this!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+⚠️❗ __The deployed version of the engineering playbook is now on [Docusaurus](https://playbook.truss.dev/).
```
Comment (rpdelaney) [inline README.md:1]: I sincerely don't know if this thought is helpful or just pedantic, so I leave it to others to decide :)

I think Docusaurus is just a markdown-to-html rendering engine, so the eng playbook is not really "on" Docusaurus?

```suggestion
⚠️❗ __The engineering playbook is now deployed on [playbook.truss.dev](https://playbook.truss.dev/) and it should be read there.
```

Comment (mikena-truss) [reply README.md:1]:
```suggestion
⚠️❗ __The deployed version of the engineering playbook is now deployed at [playbook.truss.dev](https://playbook.truss.dev/) using [Docusaurus](https://docusaurus.io/).
We recommend you read it there.
```

@rpdelaney how about this?

Comment (rogeruiz) [reply README.md:1]: I'll say that we saw this same issue with MilMove when discussing documentation. We eventually stopped calling it Docusuarus as not only is it sometimes difficult to say / spell check. It was also causing confusion as Docusaurus is a Facebook project and was causing folks to get further confused. I like the second suggestion here @mikena-truss

Comment (rpdelaney) [reply README.md:1]: Looks good!

**Pull request merged**